user-guide/enterprise-auth/intro.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using Enterprise Authentication
+description: Learn how to connect to your enterprise authentication provider, and how to leverage data from that provider in your apps.
+position: 1
+publish: true
+slug: enterprise-auth
+---
+
+# Using Enterprise Authentication
+
+NativeScript Sidekick lets you connect to existing enterprise identity and single sign-on solutions. This powerful functionality allows application developers to leverage enterprise user accounts, and to do so while avoiding the complexity of integrating these protocols into mobile apps.
+
+Let’s look at how it works.
+
+* [Step 1. Start you app](#step-1)
+* [Step 2. Select a protocol](#step-2)
+* [Step 3: Complete the configuration](#step-3)
+* [Step 4: Run your app](#step-4)
+
+<h2 id="step-1">Step 1. Start your app</h2>
+
+As a first step, [create a new app]({% slug create-app-from-template %}) using the Enterprise Auth template.
+
+![](/images/enterprise-auth/sidekick-templates.png)
```

Comment:
All images were not visible. I have fixed the image links by removing "/sidekick" from the path.